static/css/frame.scss

```diff
@@ -47,6 +47,61 @@
   }
 }
 
+.promo-panel {
+  .edit {
+    right: 50px;
+    top: 75px;
+  }
+  .promo-box {
+    border-radius: 3px;
+    box-shadow: 0 0 0 1px rgba(0, 0, 0, 0.3), 0 0 20px rgba(0, 0, 0, 0.2);
+    position: absolute;
+    width: 180px;
+    z-index: 999;
+    padding: 10px 5px;
+    text-align:center;
+  }
+
+  .title {
+    font-size: 14px;
+    line-height: 14px;
+    font-weight: bold;
+  }
+
+  .message {
+    display: inline-block;
+    font-size: 12px;
+    margin: 5px 2px;
+  }
+
+  .promo-star:before {
+    content: "\02728";
+  }
+
+  .link {
+    cursor: pointer;
+    color: #009ec0;
+  }
+
+  a.box-close{
+    float:right;
+    cursor:pointer;
+    color: #000;
+    width:15px;
+    height: 15px;
+    line-height: 10px;
+    margin: -8px -3px 0 0;  
+  }
+
+  .box-close:before {
+    content: "×";
```

Comment:
Probably better to just use the close icon from the photon design system, see the 'Navigation' section at https://design.firefox.com/icons/viewer/